packages/block-editor/src/components/block-list/style.scss

```diff
@@ -208,7 +208,6 @@
 
 	// Active entity spotlight.
 	&.has-active-entity:not(.is-focus-mode) {
-		opacity: 0.5;
```

Comment:
If we are removing the template part spotlight mode thingy, we can probably remove this class entry altogether